.travis.yml

```diff
@@ -13,25 +12,22 @@ env:
   - DB=Firebird 
   - DB=MySQL CONNECTION_STRING="Server=127.0.0.1;Uid=root;Database=nhibernate;Old Guids=True;"   
   - DB=SQLite 
-matrix:
-  allow_failures:
-    - env: DB=MySQL CONNECTION_STRING="Server=127.0.0.1;Uid=root;Database=nhibernate;Old Guids=True;"
 before_install:
-  - curl https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -
-  - curl https://packages.microsoft.com/config/ubuntu/14.04/prod.list | sudo tee /etc/apt/sources.list.d/microsoft.list
   - sudo apt-get update -qq
   - sudo apt-get install -y powershell 
   - |-
     if [[ "$DB" == "MySQL" ]]
     then
-      echo -e '[server]\nlower_case_table_names=1' | sudo tee -a /etc/mysql/my.cnf
+      echo -e '[server]\nlower_case_table_names=1\n[mysqld]\ncharacter-set-server=utf8\ncollation-server=utf8_general_ci' | sudo tee -a /etc/mysql/my.cnf
       sudo service mysql restart
     fi
   - |-
     if [[ "$DB" == "Firebird" ]]
     then
-      sudo apt-get install -y libicu-dev libtommath-dev curl
```

Comment:
`libicu-dev` and `curl` are already installed so no point to install them again.

Reply:
Why did you change it back to `libtommath-dev`? I suspect with `libtommath0`  hacky stuff below with `ln` wouldn't be required for bionic and above. Or am I wrong?

Reply:
The `libtommath-dev` references either `libtommath0` or `libtommath1` depending on the distributive. 16.04 is the last version which will include `libtommath0`. The 18.04 and above uses `libtommath1`.

Firebird 3.x is built against `libtommath0` and they refuse to rebuild it to `libtommath1`, so the `ln` will be required in 18.04 or above when using the official distribution.

However, it seems that 18.04 and above have Firebird 3.0 in the `apt`.